thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/controller.js

```diff
@@ -199,20 +171,63 @@ export default Controller.extend({
 
   /**
    * Data needed to render the stats 'cards' above the anomaly graph for this alert
+   * NOTE: buildAnomalyStats util currently requires both 'current' and 'projected' props to be present.
    * @type {Object}
    */
   anomalyStats: computed(
+    'alertEvalMetrics',
+    'isTuneAmountPercent',
+    'isTunePreviewActive',
     'customPercentChange',
     'selectedSeverityOption',
-    'alertEvalMetrics',
     'alertEvalMetrics.projected',
     function() {
       const {
-        alertEvalMetrics: evalMetrics,
-        customPercentChange: severity
-      } = this.getProperties('alertEvalMetrics', 'customPercentChange');
-      const isPercent = !this.get('selectedSeverityOption').includes('Absolute');
-      return buildAnomalyStats(evalMetrics, 'tune', severity, isPercent);
+        isTuneAmountPercent,
+        isTunePreviewActive,
+        alertEvalMetrics: metrics,
+        customPercentChange: severity,
+        selectedSeverityOption
+      } = this.getProperties(
+        'isTuneAmountPercent',
+        'isTunePreviewActive',
+        'alertEvalMetrics',
+        'customPercentChange',
+        'selectedSeverityOption'
+      );
+      const severityUnit = isTuneAmountPercent ? '%' : '';
+      const isPerfDataReady = _.has(metrics, 'current');
+      const statsCards = [
```

Comment:
I've seen this statsCard logic before. Anyway we can consolidate all uses of starsCards?

Reply:
Yes. I found a more efficient way (using `extractSeverity` method from `manage-alert-utils`.